File: FSDI 107 TaskManager - Project/wwwroot/js/task.js
class Task {
  
  constructor(title, desc, important, dueDate, alertText, location) {
    this.id = 0;
    this.title = title;
    this.description = desc;
    this.important = important;
    this.dueDate = dueDate;
    this.alertText = alertText;
    this.location = location;
    this.status = 1; // 1 = pending 2 = done

    this.user = "ThadSiwecki";
  }
}

File: FSDI 107 TaskManager - Project/Models/Task.cs
namespace TaskManager.Models 
{
    public class Task
    {
        public int Id {get; set;}

        public string Title {get; set;}

        public string Description {get; set;}

        public bool Important {get; set;}

        public string DueDate {get; set;}

        public int AlertText {get; set;}

        public string Location {get; set;}

        public int Status {get; set;}

        public string User {get; set;}

    }
}
